src/components/RichContenteditable/RichContenteditable.vue

```diff
@@ -92,20 +109,28 @@ export default {
 
 <template>
 	<div ref="contenteditable"
-		:class="{'rich-contenteditable__input--empty': isEmptyValue}"
+		v-tooltip="tooltip"
+		:class="{
+			'rich-contenteditable__input--empty': isEmptyValue,
+			'rich-contenteditable__input--multiline': multiLine,
+			'rich-contenteditable__input--overflow': isOverMaxlength,
+		}"
 		:contenteditable="contenteditable"
 		:placeholder="placeholder"
 		aria-multiline="true"
```

Comment:
I guess that this should be false (or not set, I do not know which one is better) if the `multiLine` property is false.